lib/talis-cdk-stack/talis-cdk-stack.ts

```diff
@@ -11,6 +13,23 @@ export class TalisCdkStack extends Stack {
     if (props.vpcId) {
       this.vpc = this.resolveVpc(props.vpcId);
     }
+
+    cdk.Tags.of(scope).add("tfs-app", props.app); // e.g. depot
+    cdk.Tags.of(scope).add("tfs-environment", props.deploymentEnvironment); // e.g. production staging development
+    cdk.Tags.of(scope).add("tfs-release", props.release); // e.g. 8561-105814f
+
+    // props.env comes from aws cdk core StackProps. It's optional
+    // so that an environment agnostic stack can be created.
+    // If we create an environment agnostic stack - it can not contain
+    // region or service.
+    // However, when we deploy into our environments for real - props.env will be set
+    if (props.env?.region) {
+      cdk.Tags.of(scope).add("tfs-region", props.env.region);
+      cdk.Tags.of(scope).add(
+        "tfs-service",
+        `${props.app}-${props.env.region.split("-")[0]}`
+      );
+    }
```

Comment:
This isn't formally documented anywhere, but generally, the convention is that in production we simply use `<app>-<region>` but use `<app>-<environment>-<region>` for other environments, e.g. `depot-staging-eu`, `otis-build-eu`, etc.

This has been somewhat applied to URLs we give our services, for example:
- https://files.talis.com/ - Depot server production
- https://files.ca.talis.com/ - Depot server production Canada
- https://staging-files.talis.com/ - Depot server staging